docs/figure_file_format.rst

```diff
@@ -0,0 +1,223 @@
+
+OMERO.figure files
+==================
+
+OMERO.figure files are JSON data, stored in OMERO as OriginalFiles linked to FileAnnotations.
+
+File Annotations
+----------------
+
+The FileAnnotations are identified as OMERO.figure files using a namespace of ```omero.web.figure.json```.
+
+The description of the FileAnnotation can be used to specify an Image to show as a thumbnail
```

Comment:
Interesting. I didn't know this. In terms of a larger refactoring at some point, we should probably review.

Reply:
This is an easy way to add a bit of extra info the FileAnnotation without needing to load extra annotations etc. If it's missing then we simply don't show thumbnail in File > Open list.

Reply:
Sure, it's an interesting workaround, but it probably points to a (mini-)specification of its own.